# DP: document conditional file expressions acting on the target of
# DP: symbolic links as well (except -h, -L).

--- a/bash/builtins/test.def
+++ b/bash/builtins/test.def
@@ -64,6 +64,9 @@
 
   FILE1 -ef FILE2  True if file1 is a hard link to file2.
 
+All file operators except -h and -L are acting on the target of a symbolic
+link, not on the symlink itself, if FILE is a symbolic link.
+
 String operators:
 
   -z STRING      True if string is empty.
